cc_plugin_glider/glider_dac.py

```diff
@@ -54,9 +114,8 @@ def get_ncei_table(cls, url):
 
         :param str url: the url to the NCEI table
         '''
-        resp = requests.get(url, timeout=30)
-        resp.raise_for_status()
-        return {line for line in resp.iter_lines(decode_unicode=True)}
+        fn = lambda s: set(s.splitlines())
+        return GliderCheck.request_resource(url, 'file', fn)
```

Comment:
Is the get_ncei_table function used anymore? Looks like it's deprecated in favor of ```request_resource``` If so can we just remove this function all together?

Reply:
Removed